tldw_Server_API/app/core/Embeddings/Embeddings_Server/Embeddings_Create.py

```
@@ -833,10 +840,23 @@ def __init__(
             trust_remote_code=config.trust_remote_code,
         )
 
-        self.session: Optional[ort.InferenceSession] = None
-        self.unload_timer: Optional[threading.Timer] = None
-        self.last_used_time: float = 0.0
-        self.device_providers = config.onnx_providers
+        # Capture timer and session references directly for cleanup
+        def _finalize_onnx(timer_ref, session_ref):
```

Ruff (0.14.4)

Unused function argument: `session_ref`

(ARG001)